isotovideo

```diff
@@ -206,10 +213,11 @@ sub calculate_git_hash {
     chomp($test_git_hash = qx{git rev-parse HEAD});
     $test_git_hash ||= "UNKNOWN";
     chdir($dir);
-    diag "git hash of test distribution: $test_git_hash";
+    bmwqemu::fctwarn("git hash of test distribution: $test_git_hash");
```

Comment:
Actually should be a notice instead... log4perl allows to mask different levels aside from the default ones. Will work on fixing the levels during this week